### ISSUE TITLE
User account `Edit` link disappears without 'view user accounts' permission

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#2579]
I noticed this while testing the new user permissions. But when I don't have access to VIEW a user account page, the EDIT link on the admin/people view is not available to me.

I often deny everyone access to view the user account page (since I don't want users to have profile pages) but that doesn't mean I don't have administrators who need to edit those accounts.

Let's check the permissions views is using and make sure we don't have our wires crossed.

**Comment by indigoxela** [comment]
> when I don't have access to VIEW a user account page, the EDIT link on the admin/people view is not available to me

There seems to be a misunderstanding. If someone can not _view_ something, they can not _edit_ it, too - that seems logical to me.

> I often deny everyone access to view the user account page (since I don't want users to have profile pages) but that doesn't mean I don't have administrators who need to edit those accounts.

Why shouldn't administrators - who are able to edit user accounts - be allowed to see them? :confused: I don't get it...

**Comment by klonos** [comment]
I agree that from a reasoning point of view, having the permission to edit something implies the ability to view it as well. However, I see the validity in having separate permissions for these 2 tasks. How about doing the same thing for the "edit" permissions, as we do with the "Access administration bar" permission, which when selected suggests to also (optionally) enable the "Use the administration pages" permission?

**Comment by BWPanda** [comment]
I agree that this is "by design". I think @jenlampton should create a feature request (if there isn't already one) for disabling user profile pages, as trying to use permissions for that purpose seems hacky to me...
